IntegrationTests/CarthageIntegration/CarthageIntegration.xcodeproj/project.pbxproj

```diff
@@ -447,7 +447,7 @@
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				CODE_SIGN_STYLE = Manual;
 				DEFINES_MODULE = YES;
-				DEVELOPMENT_TEAM = 8SXR2327BM;
+				DEVELOPMENT_TEAM = "";
```

Comment:
these tests are here to check that the framework can be integrated correctly by using the different package managers, and they do actually build & archive. 
This is because we've had multiple issues in the past that don't come up when running, but get surfaced when archiving. Mostly linker stuff. 
so if we remove the `development_team` here, we need to have a way to re-add it automatically when running integration tests. 

Note that these are exclusively for development